actions/promote/worker.js

```diff
@@ -49,7 +49,7 @@ async function main(params) {
             payload = 'Getting all files to be promoted.';
             updateStatusToStateLib(projectRoot, PROJECT_STATUS.IN_PROGRESS, payload, undefined, PROMOTE_ACTION);
             logger.info(payload);
-            payload = await promoteFloodgatedFiles(adminPageUri, projectExcelPath);
+            payload = await promoteFloodgatedFiles(adminPageUri, projectExcelPath, params.doPublish);
```

Comment:
nit: `doPublish` can be part of the deconstructed object here: https://github.com/adobecom/milo-fg/blob/b59911ebd4d30a05ada7dfd6fad90ac4e17c68ef/actions/promote/worker.js#L35-L37

Reply:
Thanks @sukamat, updated this.